.eslintrc.json

```diff
@@ -11,5 +11,6 @@
     "es6": true,
     "node": true,
     "jest": true
-  }
+  },
+  "ignorePatterns": ["lib/"]
```

Comment:
should we also ignore `node_modules` or is it smart enough to do that on its own.

Reply:
o rite I buried that in the [commit description](https://github.com/tvkitchen/base/pull/27/commits/a1f9e3bfb9b0ff9e8e5367cea0d50414ed51eb65). Yes, it does!

Reply:
I'm supposed to read COMMENT DESCRIPTIONS now?!?!?!!??!?!1

Reply:
Nope! My fault for not elevating that to PR description. (For real I don't think it's reasonable to assume you dug into each commit.)

Reply:
So now I'm supposed to read PR DESCRIPTIONS?!?!?!?!?!?!?!1

Reply:
Well, less "now" than "then and also always"…